tutorials/2d/custom_drawing_in_2d.rst

```diff
@@ -349,7 +349,7 @@ the same as before, except that we draw a polygon instead of lines:
         pointsArc[0] = center;
         var colors = new Color[] { color };
 
-        for (int i = 0; i <= nbPoints; i++)
+        for (int i = 1; i <= nbPoints; i++)
         {
             float anglePoint = Mathf.Deg2Rad(angleFrom + i * (angleTo - angleFrom) / nbPoints - 90);
             pointsArc[i] = center + new Vector2(Mathf.Cos(anglePoint), Mathf.Sin(anglePoint)) * radius;
```

Comment:
This misses the `anglePoint` for i = 0. I think the equivalent of the GDScript version would be something like this:

```suggestion
        for (int i = 0; i <= nbPoints; i++)
        {
            float anglePoint = Mathf.Deg2Rad(angleFrom + i * (angleTo - angleFrom) / nbPoints - 90);
            pointsArc[i + 1] = center + new Vector2(Mathf.Cos(anglePoint), Mathf.Sin(anglePoint)) * radius;
```